<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59);
background-color: rgb(21, 28, 42);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pos_Negat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Percentual">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Divide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(211, 174, 63);
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Multipl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(211, 174, 63);
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Subt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(211, 174, 63);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Equal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(41, 255, 77);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(211, 174, 63);
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Comma">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(97, 97, 104);</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
